### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by RAnders00**

### CODE REVIEW

**Review by 5E7EN — CHANGES_REQUESTED** [review]
Please keep this section safe-for-work, as I am currently sitting in my desk at work.

**Comment by RAnders00** [comment]
@5E7EN it's now safe for work.

File: README.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ Here are some ideas to get you started:
 - ⚡ Fun fact: ...
 -->
 
-🔭 I’m currently working on YOUR MOM  
-![your mom](https://cdn.discordapp.com/emojis/770060366477918208.gif?v=1)
+🔭 I’m currently working on THE WORK THAT I'VE BEEN ASSIGNED BY MY BOSS LIKE A GOOD BOY  
+![the work that i've been assigned](https://cdn.discordapp.com/emojis/770060366477918208.gif?v=1)
```
Comment (5E7EN) [inline README.md]: Sir, please swap this GIF to something more fitting the context. Perhaps an image of some nice dogs doing things that dogs do?

**Review by 5E7EN — CHANGES_REQUESTED** [review]
DLGTM

**Comment by RAnders00** [comment]
@5E7EN should be good now?

**Comment by 5E7EN** [comment]
Yes thanks for your c

**Pull request merged**